mimic/test/test_external_api.py

```diff
@@ -5,28 +5,30 @@
 from twisted.trial.unittest import SynchronousTestCase
 
 from mimic.imimic import IExternalAPIMock
-from mimic.test.dummy import ExampleAPI, ExampleExternalAPI
+from mimic.test.dummy import (
+    ExampleAPI,
+    ExampleEndpointTemplate,
+    make_example_external_api
+)
 from mimic.test.fixtures import APIMockHelper, TenantAuthentication
 
 
-class InvalidApiMock(object):
-    pass
-
-
 class TestValidationPoints(SynchronousTestCase):
 
-    def test_core_gate_check(self):
-        """
-        Test that the gate check which ensures the submitted APIs
-        are either :obj:`IAPIMock` or :obj:`IExternalAPIMock` works.
-        """
-        with self.assertRaises(TypeError):
-            APIMockHelper(self, [InvalidApiMock()])
+    def setUp(self):
+        self.eeapi_name = u"externalServiceName"
 
     def test_external_api_no_service_resource(self):
-        self.eeapi_name = u"example-api"
-        self.eeapi = ExampleExternalAPI(name=self.eeapi_name)
-        self.helper = APIMockHelper(self, [self.eeapi])
+        """
+        Validate that an external API does not provide a Resource
+        for Mimic to support that API within itself.
+        """
+        eeapi = make_example_external_api(
+            name=self.eeapi_name,
+            set_enabled=True
+        )
+        self.assertIsNotNone(eeapi)
```

Comment:
I ask because there are several test assertions and python `asserts` where this is being checked, yet it doesn't seem like returning None should be a possibility. :-)